covid_xprize/validation/cost_generator.py

```diff
@@ -8,18 +8,33 @@
 import argparse
 
 import numpy as np
+import pandas as pd
 
-from covid_xprize.validation.scenario_generator import get_raw_data
 from covid_xprize.validation.scenario_generator import NPI_COLUMNS as IP_COLUMNS
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
 FIXTURES_PATH = os.path.join(ROOT_DIR, 'data')
-DATA_FILE = os.path.join(FIXTURES_PATH, "OxCGRT_latest.csv")
+DEFAULT_GEOS = os.path.join(ROOT_DIR, '..', '..', "countries_regions.csv")
 
 
 def generate_costs(distribution='ones'):
     """
-    Returns df of costs for each IP for each geo according to distribution.
+    Returns a df of costs for each IP for default list of geos according to distribution.
+    """
+    return generate_costs_for_geos_file(DEFAULT_GEOS, distribution)
```

Comment:
Kept the original `generate_costs` API for backward compatibility (it's used by NEAT). Delegated the implementation to the methods below